.gitignore

```diff
@@ -63,3 +61,34 @@ $RECYCLE.BIN/
 ## Mac detritus #
 #################
 .DS_Store
+
+#############
+## Python
+#############
+
+*.py[co]
+
+# Packages
+*.egg
+*.egg-info
+dist/
+build/
+eggs/
+parts/
+var/
+sdist/
+develop-eggs/
+.installed.cfg
+
+# Installer logs
+pip-log.txt
+
+# Unit test / coverage reports
+.coverage
+.tox
+
+#Translations
+*.mo
+
+#Mr Developer
+.mr.developer.cfg
```

Comment:
Python files does not feel relevant to ignore, please remove together with Translations files etc. Unit test / coverage reports can stay if you wish.